config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml

```diff
@@ -0,0 +1,90 @@
+langcode: en
```

Comment:
I've noticed that we have a link like https://ec.europa.eu/commission/index_en which redirects to ec.europa.eu/info site. Should we change this URL as well?

Reply:
I don't think we need to change that, I think these redirects will be changed on the server.